SPECS/rdma-core/rdma-core.spec

```diff
@@ -304,10 +304,10 @@ rm -f %{buildroot}/%{_sbindir}/srp_daemon.sh
 %postun -n librdmacm -p /sbin/ldconfig
 
 %post -n rdma-core
-if [ -x /sbin/udevadm ]; then
-/sbin/udevadm trigger --subsystem-match=infiniband --action=change || true
-/sbin/udevadm trigger --subsystem-match=net --action=change || true
-/sbin/udevadm trigger --subsystem-match=infiniband_mad --action=change || true
+if [ -x /bin/udevadm ]; then
```

Comment:
One of the GitHub PR checks seem to be failing.
@AZaugg

Reply:
Fixed